src/single-spa-vue.js

```diff
@@ -46,14 +46,14 @@ function mount(opts, mountedInstances, props) {
   return Promise
     .resolve()
     .then(() => {
-      const otherOptions = {}
-      if (props.domElement && !opts.appOptions.el) {
-        otherOptions.el = props.domElement;
+      const appOptions = {...opts.appOptions}
```

Comment:
I found the otherOptions variable to be confusing and at the source of the bug in #16. The appOptions variable is just a shallow clone of what was given to us.